diff --git a/internal/config/config_features.go b/internal/config/config_features.go
index 18561c0..847a7f4 100644
--- a/internal/config/config_features.go
+++ b/internal/config/config_features.go
@@ -4,7 +4,7 @@ var Sponsor = Env(EnvDemo, EnvSponsor, EnvTest)
 
 // DisableWebDAV checks if the built-in WebDAV server should be disabled.
 func (c *Config) DisableWebDAV() bool {
-	if c.Public() || c.ReadOnly() || c.Demo() {
+	if c.ReadOnly() || c.Demo() {
 		return true
 	}
 
